pkg/ui/src/Slideover/Slideover.svelte

```diff
@@ -28,3 +27,20 @@
 		<slot />
 	</div>
 </div>
+
+<style>
+	@keyframes slider {
+		from {
+			transform: translateX(100%);
+			opacity: 0;
+		}
+		to {
+			transform: translateX(0px);
+			opacity: 1;
+		}
+	}
+
+	.animate-unknown {
+		animation: slider 1s ease-in-out;
```

Comment:
Can we make this 200ms? It slows down my speedrun!

On a more serious note, I agree with [this article](https://archive.is/yCKWA#39a4) (that I found after googling five seconds) that confirms my current opinions.

I also [asked ChatGPT](https://chat.openai.com/share/c3c06a66-7f29-4014-b07a-dacbe488b76c) and it also suggests between 200 and 300 ms for "simple animations".